src/app/features/home/home.page.ts

```diff
@@ -49,15 +49,8 @@ export class HomePage {
   ionViewDidEnter() {
     of(this.onboardingService.isNewLogin)
       .pipe(
-        switchTap(isNewLogin =>
-          defer(() => {
-            if (isNewLogin) {
-              return this.onboardingRedirect();
-            }
-            return VOID$;
-          })
-        ),
         concatMap(isNewLogin => this.migrationService.migrate$(isNewLogin)),
```

Comment:
Should catch errors for migrate$. If migrate$ fails, `onboardingRedirect()` still needs to go on.